lk2nd/device/dts/msm8916/msm8916-qrd-9.dts

```diff
@@ -25,5 +36,20 @@
 				compatible = "longcheer,booyi-otm1287";
 			};
 		};
+
+		
+		gpio-keys {
+			compatible = "gpio-keys";
+
+			up {
+				lk2nd,code = <0x116 /* KEY_VOLUMEUP */>;
```

Comment:
I see some needed dt-bindings :D 

I guess this one will require editing `include/dev/keys.h`, maybe move the defines to `include/dt-bindings/dev/keys.h` or something like that (outside of the lk2nd folder)